src/caret_analyze/architecture/struct/callback.py

```diff
@@ -16,10 +16,12 @@
 
 from abc import ABCMeta, abstractmethod
 
+from ...exceptions import MultipleItemFoundError
 from ...value_objects import (CallbackStructValue, CallbackType,
                               ServiceCallbackStructValue,
                               SubscriptionCallbackStructValue,
                               TimerCallbackStructValue)
+from ...value_objects.publish_topic_info import PublishTopicInfoValue
 
```

Comment:
```suggestion
from ...value_objects import (CallbackStructValue, CallbackType,
                              PublishTopicInfoValue,
                              ServiceCallbackStructValue,
                              SubscriptionCallbackStructValue,
                              TimerCallbackStructValue)

```
Do not specify submodules when import across modules.
When looking at value_object/__init__.py, it seems that `PublishTopicInfoValue` is registered.